index.bs

```diff
@@ -1770,12 +1792,25 @@ credential.
 
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialUserEntity">
     :   <dfn>id</dfn>
-    ::  The [=user handle=] of the user account entity.
+    ::  The [=user handle=] of the user account entity. To ensure secure operation, authentication and authorization
+        decisions MUST be made on the basis of this {{PublicKeyCredentialEntity/id}} member,  not the
+        {{PublicKeyCredentialEntity/displayName}} nor {{PublicKeyCredentialEntity/name}} members. See [[!RFC8266]]
```

Comment:
The class name for `{{PublicKeyCredentialEntity/id}}` and `{{PublicKeyCredentialEntity/displayName}}` should be `PublicKeyCredentialUserEntity`.

Reply:
oops, good catch, thx! fixed.